content/docs/concurrent-mode-intro.md

```diff
@@ -13,7 +13,7 @@ next: concurrent-mode-suspense.html
 
 Cette page fournit un aperçu théorique du mode concurrent. **Pour une introduction plus orientée vers la pratique, vous voudrez sans doute consulter les prochaines sections :**
 
-* [Suspense pour la récupération de données](/docs/concurrent-mode-suspense.html) décrit un nouveau mécanisme de récupération de données distantes au sein de composants React.
+* [Suspense pour le chargement de données](/docs/concurrent-mode-suspense.html) décrit un nouveau mécanisme de chargement de données distantes au sein de composants React.
```

Comment:
Pourquoi ce choix de modifier « récupérer » en « charger »? Intuitivement j’ai plutôt tendance à vouloir utiliser « récupérer » pour traduire « fetch ».

Reply:
En fait, j’aurais plutôt utiliser « charger » pour traduire « load » et donc « récupérer » pour « fetch »

Reply:
À l'usage on se rend compte que « récupération », outre son côté rapidement long / ampoulé / verbeux, n'est pas assez clair sur le côté réseau / distant qu'implique “fetch” dans un contexte web. Entre « récupération de données distantes » et « chargement de données », y'a plus photo.

Du coup j'ai homogénéisé la trad partout, comme on le fait d'habitude.